src/SprykerSdk/Spryk/Model/Spryk/Definition/Argument/Callback/EnsureInterfaceSuffixCallback.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Callback;
+
+use SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollectionInterface;
+
+class EnsureInterfaceSuffixCallback implements CallbackInterface
+{
+    public const INTERFACE_SUFFIX = 'Interface';
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'EnsureInterfaceSuffix';
+    }
+
+    /**
+     * @param \SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollectionInterface $argumentCollection
+     * @param mixed|null $value
+     *
+     * @return mixed
+     */
+    public function getValue(ArgumentCollectionInterface $argumentCollection, $value)
+    {
+        if (mb_substr($value, - mb_strlen(static::INTERFACE_SUFFIX)) !== static::INTERFACE_SUFFIX) {
+            $value = $value . static::INTERFACE_SUFFIX;
+        }
+
+        return ucfirst($value);
```

Comment:
The same as above. Please also note that ucfirst returns string and doc block says `mixed`. Please also adjust return type.

Reply:
Return type is required by the interface the class implements. To make sure `ucfirst` takes string, I cast to string explicitly.